src/ui/theme.h

```diff
@@ -56,6 +56,8 @@ namespace ui {
     static int getStyleFlagsForWidget(const Widget* widget);
   };
 
+  static ui::Style m_defaultStyle(nullptr);
```

Comment:
Move this global variable inside `Theme` class, making this a static member variable of `Theme`.

As it's now there will be multiple `m_defaultStyle` variables (one for each `#include "ui/theme.h"` we do).